__tests__/api.ts

```diff
@@ -59,11 +72,15 @@ describe('fetchApi()', () => {
   })
 
   test('transfers meta data to fetch()', () => {
+    serverMock('https://api.serlo.org/', returnResponseText('<api-result>'))
+
```

Comment:
This need to be moved into beforeAll(). Note, that the result should be `{ "result": 42 }`.